HelpMyStreetFE/HelpMyStreetFE/Views/Registration/StepThree.cshtml

```diff
@@ -15,104 +15,31 @@
     ViewData["Title"] = "Volunteers - Registration";
     Layout = "~/Views/Shared/_LayoutNoHeader.cshtml";
 
-    var volunteerOptions = new List<SupportActivityViewModel>();
+    var volunteerOptions = Model.ActivityDetails
```

Comment:
I think we can lose this block of code...  (As a general rule, code in Views should be avoided.)
* OrderBy can be done as part of the foreach below, or in the Controller if preferred.
* SupportActivityViewModel essentially duplicates SupportActivityDetail, so can be dropped.  (If we did want to keep it for separation of concerns, the better place to have the Select would be in the Controller.)
* Case where count is zero should never be hit.  (If we're nervous that it could be, add a check in the Controller - but throw an exception rather than showing default options.)